layouts/shortcodes/pipelines/latest-version.html

```diff
@@ -1 +1 @@
-0.5.1
\ No newline at end of file
+1.0.0
```

Comment:
Why version 1.0.0 and not the latest?

Reply:
https://github.com/kubeflow/pipelines 1.0.0 seams to be the latest one to me